mockserver/public/v4/state

```diff
@@ -10048,8 +10048,8 @@
         "source": "demanda.ree.es",
         "stateDatetime": "2022-01-04T08:00:00Z",
         "storage": {
-          "battery": -0,
-          "hydro": -0
+          "battery": 0,
+          "hydro": 0
```

Comment:
While these changes makes sense, it's not what our app-backend serves - so I'm not sure if we really want to do this "cleaning"?

Reply:
This is caused by Python not respecting these negative 0 values when formatting the JSON string. The negative 0 doesn't have any semantic meaning in our system, but is an effect of the app-backend `negate_if_not_none` function.